github/ci/prow/files/config.yaml

```diff
@@ -60,6 +60,7 @@ tide:
     kubevirt/kubectl-virt-plugin: squash
     kubevirt/node-maintenance-operator: squash
     kubevirt/ansible-kubevirt-modules: squash
+    kubevirt/hyperconverged-cluster-operator: merge
```

Comment:
Is it a request of the project to use `merge` instead of `squash` like everyone else?

Reply:
you can use whatever HCO maintainers prefer.

Reply:
no, on the project itself we are using squash

Reply:
fixed